projects/1.19/assets/varied-adventure/va/lang/zh_cn.json

```
@@ -0,0 +1,251 @@
+{
+  "advancements.bleed.descr": "受到诅咒伤口的负面效果。",
+  "advancements.bleed.title": "流血！",
+  "advancements.call_a_monster.descr": "获得一个蛇笛。",
+  "advancements.call_a_monster.title": "召唤一只怪物",
+  "advancements.cursed_gear.descr": "获得所有具有诅咒伤害效果的武器",
+  "advancements.cursed_gear.title": "诅咒齿轮",
+  "advancements.eye.descr": "杀死一只梦魇观察者",
+  "advancements.eye.title": "审视",
+  "advancements.give_me_your_soul.descr": "获得灵魂之光.",
+  "advancements.give_me_your_soul.title": "将你的灵魂交于我！",
+  "advancements.heart_of_the_nightmares.descr": "获得一个梦魇之心。",
+  "advancements.heart_of_the_nightmares.title": "惊悸不安",
+  "advancements.is_that_an_amber.descr": "找到一块琥珀。",
+  "advancements.is_that_an_amber.title": "那是琥珀吗？",
+  "advancements.make_scale_armor.descr": "穿戴全套鳞板盔甲。",
+  "advancements.make_scale_armor.title": "海怪装甲",
+  "advancements.no_more_poison.descr": "喝下抗毒药剂",
+  "advancements.no_more_poison.title": "百毒不侵",
+  "advancements.skull.descr": "杀死一只物种守护神",
+  "advancements.skull.title": "薨髑",
+  "advancements.snake.descr": "杀死一只巴兹里斯克蛇。",
+  "advancements.snake.title": "传说之蛇",
+  "advancements.the_execution.descr": "在塔顶杀死一名骑士。",
+  "advancements.the_execution.title": "行刑",
+  "advancements.this_is_the_power.descr": "使用神力图腾。",
+  "advancements.this_is_the_power.title": "这就是力量！",
+  "advancements.two_times_stronger.descr": "使用气血图腾。",
+  "advancements.two_times_stronger.title": "血气方刚",
+  "advancements.varied_adventure_advancements.descr": "欢迎来到多样化的冒险！",
+  "advancements.varied_adventure_advancements.title": "多样化的冒险",
+  "block.v_a.moerolite_block": "莫洛利特块",
+  "block.v_a.moerolite_ore": "莫洛利特矿石",
+  "block.v_a.raw_moerolite_block": "莫洛利特原矿块",
```

```suggestion
  "block.v_a.raw_moerolite_block": "粗莫洛利特块",
```
见原版`粗铁`/`Raw Iron`
话说moreolite有啥词源吗（）

---

moreolite是音译的，没有找到词源